src/client/UserModel.ts

```diff
@@ -936,46 +940,55 @@ export class UserModel implements ConsoleErrorLogger {
         lockEnd: "0",
         balance: "0",
         claimable: "0",
-        claimableETH: "0",
-        totalClaimableToken: "0",
-        totalClaimableETH: "0",
+        claimableV2: "0",
+        claimableETHV2: "0",
+        totalClaimableTokenV2: "0",
+        totalClaimableETHV2: "0",
         dillApproval: "0",
       };
     }
   }
   async getUserDillStats(): Promise<IUserDillStats> {
     const dillContractAddr: string = ADDRESSES.get(ChainNetwork.Ethereum).dill;
+    const feeDistributorAddrV2: string = ADDRESSES.get(
+      ChainNetwork.Ethereum,
+    ).feeDistributorV2;
     const feeDistributorAddr: string = ADDRESSES.get(
       ChainNetwork.Ethereum,
     ).feeDistributor;
-
     const dillContract: Dill = Dill__factory.connect(
       dillContractAddr,
       this.providerFor(ChainNetwork.Ethereum),
     );
-
-    const feeDistributorContract: FeeDistributorV2 = FeeDistributorV2__factory.connect(
+    const feeDistributorContract: FeeDistributor = FeeDistributor__factory.connect(
       feeDistributorAddr,
       this.providerFor(ChainNetwork.Ethereum),
     );
+    const feeDistributorContractV2: FeeDistributorV2 = FeeDistributorV2__factory.connect(
+      feeDistributorAddrV2,
+      this.providerFor(ChainNetwork.Ethereum),
+    );
 
     const pickleContract: Erc20 = Erc20__factory.connect(
       ADDRESSES.get(ChainNetwork.Ethereum).pickle,
       this.providerFor(ChainNetwork.Ethereum),
     );
-
+    
+    const claimable = await feeDistributorContract.callStatic["claim(address)"](this.walletId, {
+      gasLimit: 1000000,
+    });
     const [lockStats, balance, userClaimable, allowance] = await Promise.all([
       dillContract.locked(this.walletId, { gasLimit: 1000000 }),
       dillContract["balanceOf(address)"](this.walletId, { gasLimit: 1000000 }),
-      feeDistributorContract.callStatic["claim(address)"](this.walletId, {
+      feeDistributorContractV2.callStatic["claim(address)"](this.walletId, {
         gasLimit: 1000000,
       }),
       pickleContract.allowance(this.walletId, dillContractAddr),
     ]);
 
     let totalClaimable;
     try {
-      totalClaimable = await feeDistributorContract.callStatic["claim_all(address)"](this.walletId, {
+      totalClaimable = await feeDistributorContractV2.callStatic["claim_all(address)"](this.walletId, {
```

Comment:
This could also be included in the above `Promise.all()` with a `catch` statement, setting the amount to `[0,0]` on error. You would then include an if statement checking if `userClaimable[0] > 0` and `totalClaimable[0] == 0`, which would tell you it errored, and so you need to set `totalClaimable = userClaimable`

Reply:
Fixed.